docs/source/fetching/persisted-queries.mdx

```diff
@@ -0,0 +1,98 @@
+---
+title: Persisted Queries
+---
+
+Apollo supports features called "Persisted Queries" and "Automatic Persisted Queries" (APQs), to avoid sending the same large query documents over and over, and to allow safelisting of queries.
+
+Each query or mutation is identified by the SHA256 hash of its contents that can be sent to the server in lieu of sending a full query document. For APQs specifically, if the hash can't be found by the server, it sends back an error indicating that it needs the full query. If it receives this specific error, Apollo iOS will automatically retry the operation with the full query document without you having to do anything.
+
+### Support
+
+| | **Persisted Queries** | **Automatic Persisted Queries** |
+| - | :---------------: | :-------------------------: |
+| **GraphOS** | Yes | No |
+| **Apollo Server** | Yes?, version ?? | Yes, version ?? |
+
+### Feature
+
+| | **Persisted Queries** | **Automatic Persisted Queries** |
+| - | :---------------: | :-------------------------: |
+| **Pre-registered queries** | Yes | Yes |
+| **Automatically persist queries from client** | No | Yes |
+| **Allow only safelisted persisted queries** | Yes | Yes |
```

Comment:
To me this is a little confusing - would it be better to say that persisted queries' default implementation is compatible with APQs and then additional safelisting requires APQs? Though yes, I suppose you can have safelisting of _documents_ and still use APQs. Is that what you're going for in this table?

Reply:
So this table I was just trying to see if I could break down some of the key features or callouts between the two, definitely open to suggestions for changes or another way to represent the differences especially if this is confusing.

Reply:
I would either use [this table](https://deploy-preview-496--apollo-monodocs.netlify.app/graphos/operations/persisted-queries#differences-from-automatic-persisted-queries), or link to it with a slight introduction.

(BTW this is entire doc, table included, is still WIP and I'm happy to change it.)